sklearn/neighbors/nca.py

```diff
@@ -301,7 +300,7 @@ def _validate_params(self, X, y):
         # Check the preferred dimensionality of the projected space
         if self.n_components is not None:
             check_scalar(self.n_components, 'n_components',
-                         integer_types, 1)
+                         int, 1)
```

Comment:
This could probably fit in one line now:

```python
            check_scalar(self.n_components, 'n_components', int, 1)
```

Reply:
Done! :)